svn_select_to_git.py

```diff
@@ -892,6 +892,124 @@ def processRevision(exportDir, gitDir, log):
   # End if
 # End def processRevision
 
+############################################
+###
+### SVN Externals -> Externals_CAM functions
+###
+############################################
+
+def ReadExternalList(filename):
+    """Reads in text file list of CAM external directories/files"""
+
+    #Check if file exists:
+    if not os.path.exists(filename):
+        perr("External list, '{}', does not exist".format(filename))
+
+    ext_list = {}
+    try:
+        with open(filename) as f:
+            #Read in each line separately:
+            ext_list = f.readlines()
+     
+            #Remove any white space and endlines:
+            ext_list = [x.strip() for x in ext_list]                                
+
+    #Catch any errors, if present: 
+    except ValueError as e:
+        perr(e)
```

Comment:
Not sure what the value is of a separate catch for this if you do not plan on handling it (it will be caught below).